tests/graph-container.js

```diff
@@ -18,8 +18,11 @@ describe('@graph container', () => {
         "value": "x"
       }
     };
-    jsonld.expand(doc, (err, expanded) => {
-      assert.ifError(err);
+    const p = jsonld.expand(doc);
+    assert(p instanceof Promise);
+    p.catch(e => {
```

Comment:
Could change many/all of these tests to async functions and return a promise.  Avoids needing to call done().

Reply:
@davidlehn, yup, but was just copying the convention in the file. Another PR could do that clean up later.